server/src/JWT.hh

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <jwt-cpp/jwt.h>
+#include <jwt-cpp/traits/nlohmann-json/defaults.h>
+#include <oatpp/web/server/handler/AuthorizationHandler.hpp>
+
+#include "Config.hh"
+
+namespace server {
+
+using oatpp::Void, oatpp::Any, oatpp::String, oatpp::Int8, oatpp::UInt8,
+  oatpp::Int16, oatpp::UInt16, oatpp::Int32, oatpp::UInt32, oatpp::Int64,
+  oatpp::UInt64, oatpp::Float32, oatpp::Float64, oatpp::Boolean, oatpp::Object,
+  oatpp::Enum, oatpp::Vector, oatpp::List, oatpp::UnorderedSet,
+  oatpp::UnorderedMap;
```

Comment:
If you want to use `using` then please replace it with:
```c++
using namespace oatpp::data::mapping::type;
using namespace server::dto;
```

Reply:
clang tidy disallows using namespaces as you probably never use all members of the namespace

Reply:
> clang tidy disallows using namespaces

Some clang-tidy checks shouldn't be taken as a 'rule' but more of a 'recommendation'. Al least that's how I view them.

> as you probably never use all members of the namespace

We already don't use most of the things that are included here so it won't make much difference. Mind that I don't want to include all of `oatpp` namespace, but only the type aliases.

Reply:
lmao
```
error: ‘dto’ is not a namespace-name
   12 | using namespace server::dto;
```

Reply:
And this only happens here, in `JWT.hh` - not in other places (eg `PlaceController`)

Reply:
Oh, we don't need `server::dto` namespace in this file. I did include it because I saw you did similar thing with all the dtos in a different file.

Reply:
Yes, the dtos aren't included so `server::dto` is not a 'namespace-name', because this name doesn't exist in this file at all.